rockcraft.yaml

```diff
@@ -18,16 +18,16 @@ platforms:  # The platforms this ROCK should be built on and run on
   amd64:
 
 environment:
-  JAVA_HOME: /usr/lib/jvm/java-17-openjdk-amd64
+  JAVA_HOME: /usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
**question:** Why 8? Just to match VM? If so, why 8 on VM?

Reply:
Yes. If we want to move to 17 that's fine, but we should do that consistently, in VM and K8s

Reply:
Is it not better to upgrade the VM version instead of downgrading this java version?